lib/naughty/core.lua

```diff
@@ -364,6 +360,27 @@ function naughty.destroy(notification, reason, keep_visible)
     end
 end
 
+--- Destroy all notifications on given screens.
+--
+-- @param screens Screens on which notifications should be destroyed. If nil,
+-- destroy notifications on all screens.
+-- @param reason One of the reasons from notificationClosedReason.
```

Comment:
Done. Also, I added return value to be consistent with naughty.destroy function.